runtime/gc_base/GCExtensions.cpp

```diff
@@ -338,3 +341,19 @@ MM_GCExtensions::needScanStacksForContinuationObject(J9VMThread *vmThread, j9obj
 #endif /* JAVA_SPEC_VERSION >= 19 */
 	return needScan;
 }
+
+bool
+MM_GCExtensions::isCurrentScavengerPhaseConcurrent(MM_EnvironmentBase* env)
+{
+	bool isConcurrent = false;
+#if defined(OMR_GC_CONCURRENT_SCAVENGER)
+	if (MUTATOR_THREAD == env->getThreadType()) {
+		J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
+		isConcurrent = 0 != currentThread->readBarrierRangeCheckTop;
```

Comment:
even here we could rely on isConcurrentScavengerInProgress (for the same reasons as in preMountContinuation)